### PR TITLE
1526-Config-changes-type-dropdown-multi-select

### LINKED ISSUE #1526: Config changes type dropdown multi-select
Using new api from https://github.com/flanksource/mission-control/issues/875

Use a tristate toggle to create MatchItems style

### PR DESCRIPTION
**Pull request by mainawycliffe**
Closes #1526

### CODE REVIEW

**Comment by moshloop** [comment]
@mainawycliffe The styling is also not consistent with the other dropdowns

**Comment by mainawycliffe** [comment]
Cool, will do

**Comment by mainawycliffe** [comment]
- [ ] Fix dropdown to clear on toggle

File: src/ui/TristateReactSelect/TristateReactSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+          return (
+            <>
+              <div
+                data-tooltip-id={`item-${v.id}`}
+                data-tooltip-content={`${
+                  +value === 1 ? "Include" : "Exclude"
+                } ${v.label}`}
+                key={v.label}
+                className={clsx(
+                  "w-auto flex flex-row items-center gap-1 px-2 py-1 max-w-full overflow-hidden text-ellipsis text-nowrap bg-gray-400 text-white"
```
Comment (moshloop) [inline src/ui/TristateReactSelect/TristateReactSelect.tsx:178]:
```suggestion
                  "w-auto flex flex-row items-center gap-1 px-2 py-1 max-w-full overflow-hidden text-ellipsis text-nowrap bg-gray-200"
```

**Pull request merged**